pvfactors/tests/test_viewfactors/test_data/__init__.py

```diff
@@ -1,63 +1,68 @@
-        0., 0., 0., 0., 0., 0., 0., 0.,
-        0., 0., 0., 0.974],
-     [0.026, 0.03, 0.081, 0.224, 0.104, 0.088, 0., 0.012,
-        0.047, 0.338, 0., 0., 0., 0., 0.027, 0.,
-        0., 0., 0., 0.022],
-     [0., 0., 0., 0., 0., 0., 0., 0.,
-        0., 0., 0., 0., 0., 0., 0., 0.,
-        0., 0., 0., 0.]])
+    [[0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0.006, 0., 0.001, 0.001, 0.,
+        0.001, 0.992],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0.196, 0., 0.08, 0.049, 0.,
+        0.021, 0.654],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0.003, 0., 0.097, 0.164, 0.,
+        0.095, 0.64],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0., 0., 0.034, 0.066, 0.,
+        0.18, 0.719],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0.001, 0.001, 0.001, 0., 0., 0.001, 0.002, 0.,
+        0.262, 0.733],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0.001, 0.001, 0.001, 0., 0.002, 0., 0., 0.,
+        0.035, 0.961],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0., 0., 0., 0., 0.001,
+        0., 0.999],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0.317, 0., 0.027, 0.017, 0.,
+        0.011, 0.627],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0.026, 0., 0.165, 0.152, 0.,
+        0.044, 0.612],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0.001, 0.001, 0.001, 0., 0., 0.01, 0.016, 0.,
+        0.317, 0.654],
+     [0., 0., 0., 0., 0.002, 0.005, 0.008, 0., 0.,
+        0.002, 0., 0., 0., 0., 0., 0.011, 0.007, 0.,
+        0., 0.965],
+     [0., 0., 0., 0.001, 0.002, 0.005, 0.011, 0., 0.,
+        0.002, 0., 0., 0., 0., 0., 0.016, 0.009, 0.,
+        0., 0.954],
+     [0., 0., 0., 0.001, 0.002, 0.005, 0.015, 0., 0.,
+        0.003, 0., 0., 0., 0., 0., 0.025, 0.013, 0.,
+        0., 0.936],
+     [0.315, 0.282, 0.003, 0., 0., 0., 0., 0.338, 0.028,
+        0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0.035],
+     [0., 0., 0., 0., 0., 0.005, 0.016, 0., 0.,
+        0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0.027, 0.952],
+     [0.071, 0.231, 0.167, 0.04, 0.002, 0., 0., 0.058, 0.351,
+        0.021, 0.007, 0.011, 0.017, 0., 0., 0., 0., 0.,
+        0., 0.025],
+     [0.065, 0.141, 0.282, 0.076, 0.003, 0., 0., 0.036, 0.324,
+        0.035, 0.004, 0.006, 0.009, 0., 0., 0., 0., 0.,
+        0., 0.019],
+     [0., 0., 0., 0., 0., 0., 0.026, 0., 0.,
+        0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0.974],
+     [0.026, 0.03, 0.081, 0.104, 0.224, 0.088, 0., 0.012, 0.047,
+        0.338, 0., 0., 0., 0., 0.027, 0., 0., 0.,
+        0., 0.022],
+     [0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0., 0., 0., 0., 0., 0., 0., 0.,
+        0., 0.]])
+
+vf_left_cut_sum_axis_one_rounded = np.array(
+    [0.009, 0.346, 0.359, 0.28, 0.268, 0.04, 0.001, 0.372, 0.387,
+     0.346, 0.035, 0.046, 0.064, 0.966, 0.048, 0.976, 0.981, 0.026,
+     0.977, 0.])
```

Comment:
I checked that the values of this vector stayed consistent when I updated the matrix above (which was updated because of the new surface indexing of the new class implementation)